open-policy-agent/infrastructure-general/namespace-regulation-for-pods/template.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: templates.gatekeeper.sh/v1beta1
+kind: ConstraintTemplate
+metadata:
+  name: k8snamespaceregulation
+  annotations:
+    description: Disallows pods that request to run in specific namespaces, the list of disallowed namespaces provided as a parameter array from the constraint
+spec:
+  crd:
+    spec:
+      names:
+        kind: k8sNamespaceRegulation
+      validation:
+        # Schema for the `parameters` field
```

Comment:
We should remove parameters if we end up using `namespaces` instead of the parameters we are using now